core/client/app/components/gh-posts-list-item.js

```diff
@@ -34,10 +34,6 @@ export default Component.extend(ActiveLinkWrapper, {
         return Ember.String.htmlSafe(`background-image: url(${this.get('authorAvatar')})`);
     }),
 
-    viewOrEdit: computed('previewIsHidden', function () {
-        return this.get('previewIsHidden') ? 'editor.edit' : 'posts.post';
-    }),
-
```

Comment:
Rather than calculate this view logic for every item, just threw an inline if into the template.